js/custom.js

```diff
@@ -17,54 +17,48 @@ function toggleAnswer(answerId, element) {
 // Counter Animation of the home page 
 document.addEventListener('DOMContentLoaded', () => {
     const counters = document.querySelectorAll('.count');
-
     // Function to animate the counter
     function animateCounter(counter) {
         const target = +counter.getAttribute('data-target');
-        const speed = 200;
+        const speed = 200; 
 
         const updateCount = () => {
-            const current = +counter.innerText;
-            const increment = target / speed;
+            const current = +counter.innerText.replace(/,/g, '');
+            const increment = Math.ceil(target / speed);
 
             if (current < target) {
-                counter.innerText = Math.ceil(current + increment);
-                setTimeout(updateCount, 10); // Repeat every 10ms
+                counter.innerText = Math.min(current + increment, target).toLocaleString();
+                setTimeout(updateCount, 10);
+            } else if (target === 170) {
+                counter.innerText = target.toLocaleString();
             } else {
-
-                if (target === 170) {
-                    counter.innerText = `${target}`;
-                } else {
-                    counter.innerText = `${target}+`;
-                }
-
+                counter.innerText = target.toLocaleString() + '+';
```

Comment:
You have included changes relating to the counters pull request.  Are you trying to sneak these past?  :grin: